pkg/controller/controller_clusterservicebroker_test.go

```diff
@@ -286,6 +286,19 @@ func TestReconcileClusterServiceBrokerExistingServiceClassAndServicePlan(t *test
 	assertNumberOfActions(t, kubeActions, 0)
 }
 
+func TestReconcileClusterServiceBrokerSetOSBTimeOut(t *testing.T) {
```

Comment:
there is a pattern that each test have a description, e.g. https://github.com/kubernetes-sigs/service-catalog/blob/48e6805363476ffb91faabf6aca6d03c96d123b0/pkg/controller/controller_clusterservicebroker_test.go#L692-L695

so please add sth similar also here

Reply:
ok, I will recheck it, thanks.